### PR TITLE
[crossdock-lib PART3] - Move behavior package into crossdock-go

### PR DESCRIPTION
**Pull request by breerly**
@yarpc/golang

### CODE REVIEW

File: crossdock-go/crossdock/doc.go [hunk 1]
```diff
@@ -34,4 +34,5 @@
 // 	entries := behavior.Run(func(s Sink) {
 // 		MyBehavior(s, someParams)
 // 	})
-package behavior
+
```
Comment (abhinav) [inline crossdock-go/crossdock/doc.go]: Delete this empty line. The blob above is the package documentation so it should be attached to the `package` declaration.

Comment (abhinav) [inline crossdock-go/crossdock/doc.go]: Also, consider updating or deleting the examples in the docblock above.

Comment (breerly) [inline crossdock-go/crossdock/doc.go]: That's a thing? Got it 👍

**Comment by abhinav** [comment]
👍

**Pull request merged**